src/modules/vmount/vmount_params.c

```diff
@@ -228,3 +228,13 @@ PARAM_DEFINE_FLOAT(MNT_OFF_ROLL, 0.0f);
 * @group Mount
 */
 PARAM_DEFINE_FLOAT(MNT_OFF_YAW, 0.0f);
+
+/**
+* Multiplier for the degrees in MAVLink output mode.
```

Comment:
I would like this to be easier for a user. What about calling the param `MNT_OUT_CUSTOM` and then an enum option `GREMSY_PIXY_F` and that would internally mean that a factor of 100 is used. Otherwise, it's very cryptic why this param exists or what it does.

Reply:
Great idea! See latest commit.